public/docs/_examples/dependency-injection/dart/web/main_1.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:angular2/platform/browser.dart';
 import 'package:dependency_injection/app_component.dart';
 import 'package:dependency_injection/heroes/hero_service.dart';
-//#docregion bootstrap
 
 main() {
-// Injecting services in bootstrap works but is discouraged
-  bootstrap(AppComponent, [HeroService]);
-//#enddocregion bootstrap
-}
\ No newline at end of file
+  //#docregion bootstrap
+  bootstrap(AppComponent,
+           [HeroService]); // DISCOURAGED (but works)
```

Comment:
Would be nicer to explain the purpose of the providers list on `bootstrap` instead. I.e. to override core features/services provided by Angular, etc.